### ISSUE TITLE
Set learning late for transformer base with 50 batch size and update freq of 1

### ISSUE THREAD

**Issue by ajesujoba** [pytorch/fairseq#2180]
I’m training a NMT(en-fr, transformer, base) on a single GPU(12gb) with a batch size of 50 (max length of source and target = 60) and update_freq=1. Please do you have an idea of what possible(most appropriate) value I should initialize my learning rate based on the batch size and update frequency?

Note: Average wpb = 1300

I am actually implementing the method in this paper (in fairseq - hack): https://www.aclweb.org/anthology/P19-1178.pdf . So batches are generated in the fly.
